proto/zitadel/settings/v2alpha/settings_service.proto

```diff
@@ -0,0 +1,343 @@
+            auth_option: {
+                permission: "policy.read"
+            }
+        };
+
+        option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+            summary: "Get the current active lockout settings";
+            description: "Return the current active lockout settings for the requested context"
+            responses: {
+                key: "200"
+                value: {
+                    description: "OK";
+                }
+            };
+        };
+    }
+}
+
+message GetLoginSettingsRequest {
+    zitadel.object.v2alpha.RequestContext ctx = 1;
+}
+
+message GetLoginSettingsResponse {
+    zitadel.object.v2alpha.Details details = 1;
+    zitadel.settings.v2alpha.LoginSettings settings = 2;
+}
+
+message GetPasswordComplexitySettingsRequest {
+    zitadel.object.v2alpha.RequestContext ctx = 1;
+}
+
+message GetPasswordComplexitySettingsResponse {
+    zitadel.object.v2alpha.Details details = 1;
+    zitadel.settings.v2alpha.PasswordComplexitySettings settings = 2;
+}
+
+message GetBrandingSettingsRequest {
+    zitadel.object.v2alpha.RequestContext ctx = 1;
+}
+
+message GetBrandingSettingsResponse {
+    zitadel.object.v2alpha.Details details = 1;
+    zitadel.settings.v2alpha.BrandingSettings settings = 2;
+}
+
+message GetDomainSettingsRequest {
+    zitadel.object.v2alpha.RequestContext ctx = 1;
+}
+
+message GetDomainSettingsResponse {
+    zitadel.object.v2alpha.Details details = 1;
+    zitadel.settings.v2alpha.DomainSettings settings = 2;
+}
+
+message GetLegalSettingsRequest {
+    zitadel.object.v2alpha.RequestContext ctx = 1;
+}
+
+message GetLegalSettingsResponse {
+    zitadel.object.v2alpha.Details details = 1;
+    zitadel.settings.v2alpha.LegalSettings settings = 2;
+}
+
+message GetLockoutSettingsRequest {
+    zitadel.object.v2alpha.RequestContext ctx = 1;
+}
+
+message GetLockoutSettingsResponse {
+    zitadel.object.v2alpha.Details details = 1;
+    zitadel.settings.v2alpha.LockoutSettings settings = 2;
+}
+
+message GetActiveIdentityProvidersRequest {
+    zitadel.object.v2alpha.RequestContext ctx = 1;
+}
+
+message GetActiveIdentityProvidersResponse {
+    zitadel.object.v2alpha.Details details = 1;
+    repeated zitadel.settings.v2alpha.IdentityProvider identity_providers = 2;
+}
+
+message GetGeneralSettingsRequest {}
+
+message GetGeneralSettingsResponse {
+    string default_org_id = 1 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "default organization for the current context"
+        }
+    ];
+    string default_language = 2 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "default language for the current context"
+            example: "\"en\""
+        }
+    ];
+    repeated string supported_languages = 3 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            example: "[\"en\", \"de\", \"it\"]"
+        }
+    ];
+}
```

Comment:
why are those not in a separat file?

Reply:
i would leave it as is as we don't really request the projections and cannot return a details object
moving those 3 fields into a separate message is to much IMO

Reply:
👍🏻